src/types.ts

```diff
@@ -129,28 +129,34 @@ export interface FuseChannel {
 /** If `FuseChannel.rgbColor` is this value, it is disabled from fusion. */
 export const FUSE_DISABLED_RGB_COLOR = 0;
 
-/**
- * Provide options to control the visual appearance of a Volume
- * @typedef {Object} VolumeChannelDisplayOptions
- * @property {boolean} enabled array of boolean per channel
- * @property {Array.<number>} color array of rgb per channel
- * @property {Array.<number>} specularColor array of rgb per channel
- * @property {Array.<number>} emissiveColor array of rgb per channel
- * @property {number} glossiness array of float per channel
- * @property {boolean} isosurfaceEnabled array of boolean per channel
- * @property {number} isovalue array of number per channel
- * @property {number} isosurfaceOpacity array of number per channel
- * @example let options = {
-   };
- */
 export interface VolumeChannelDisplayOptions {
+  /** Whether the channel's volume data should be rendered for this channel. */
   enabled?: boolean;
+  /** RGB color array, with values in the range of [0, 255]. */
   color?: [number, number, number];
+  /**
+   * RGB color array for specular (highlight) color, with values in the range of
+   * [0, 255].
+   */
   specularColor?: [number, number, number];
+  /**
+   * RGB color array for emissive (glow) color, with values in the range of [0,
+   * 255].
+   */
   emissiveColor?: [number, number, number];
+  /**
+   * Exponent factor controlling the glossiness ("shininess") of the material. 0
+   * is default.
+   */
```

Comment:
nit: did a formatter want these 80 wide? would love to have them on one line if possible (this is wildly unimportant but I'm always a fan of a single-line doc comment when possible)

Reply:
Yeah, it's a manual formatter though so I can just undo it.